src/clj/game/core/diffs.clj

```diff
@@ -65,7 +74,7 @@
      (if (get-in @state [:options :spectatorhands])
        (assoc @state :corp corp-deck :runner runner-deck)
        (assoc @state :corp corp-private :runner runner-private))
-     (assoc @state :corp corp-deck :runner runner-deck)]))
+     @state]))
```

Comment:
What's this change do?

Reply:
I first fixed the broken behavior with the change in `set_up.clj:130`. Now the initial state fits to the diffs that I save. But in this correct initial state the cards in the decks are not face-up, since that's also not the way it's done in game.

`corp-private` contains closed info for both R&D and HQ. `corp-deck` is the same, but with open info for HQ. R&D is only open info if `:view-deck` is set in `(:corp @state)`.

Therefore to get open info for R&D all the time, I need to grab the state before any info gets stripped, and that's simply just `@state`. That fourth entry of `private-states` is then available under `(:hist-state (public-states @state))` after getting stripped of other sensible information like the blocked-users list.

Reply:
excellent, thanks